src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -521,32 +523,31 @@ public void go() {
             runtimeError = "REVERT opcode executed";
           }
         } else {
-          getResult().getInternalTransactions().add(0,rootInternalTransaction);
           deposit.commit();
         }
       } else {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this "else" keyword to a new dedicated line. ([squid:RightCurlyBraceDifferentLineAsNextBlockCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2wdPPRVdwyhXM7&open=AWak_C2wdPPRVdwyhXM7)

[ISSUE_KEY]: AWak_C2wdPPRVdwyhXM7